test/module/irohad/multi_sig_transactions/storage_test.cpp

```diff
@@ -13,12 +13,23 @@ using namespace iroha;
 
 auto log_ = logger::log("MstStorageTest");
 
+class StorageTestCompleter : public DefaultCompleter {
```

Comment:
Looks like code duplication. I suggest to refactor it with one class in a shared header file.